src/app.py

```diff
@@ -131,7 +134,32 @@ def process_workflow_job():
     return True
 
 
+@scheduler.task('interval', id='monitor_queued', seconds=30)
+def monitor_queued_jobs():
+    """ Return the job that has been queued and not starting for long time. """
+    app.logger.debug("Starting monitor_queued_jobs")
+    if not jobs:
+        return
+
+    old_queued_job = min(jobs.items(), key=lambda x: x[1])
+    job_id, time_start = old_queued_job
```

Comment:
```suggestion
    job_id, time_start = min(jobs.items(), key=lambda x: x[1])
```

Reply:
this 2-line is to make the code readable, but I'm ok in both ways.